lib/rucio/core/replica.py

```diff
@@ -1916,9 +1997,11 @@ def __cleanup_after_replica_deletion(scope_name_temp_table, scope_name_temp_tabl
 
     # Get collection replicas of collections which became empty
     if clt_replicas_to_delete:
-        session.query(scope_name_temp_table).delete()
+        stmt = delete(scope_name_temp_table)
+        session.execute(stmt)
         session.execute(insert(scope_name_temp_table), [sn._asdict() for sn in clt_replicas_to_delete])
```

Comment:
Re: https://github.com/rucio/rucio/pull/6591/files#r1549891959

Reply:
Changed to:

```
        values = [sn._asdict() for sn in clt_replicas_to_delete]
        stmt = insert(scope_name_temp_table)
        session.execute(stmt, values)
```